godtools/App/Features/Onboarding/Models/OnboardingQuickStartItem.swift

```diff
@@ -13,4 +13,5 @@ struct OnboardingQuickStartItem {
     var title: String
     var linkButtonTitle: String
     var linkButtonFlowStep: FlowStep
+    var linkButtonAnalyticsAction: String
```

Comment:
Hey @reldredge71 I didn't catch this before, but can you change these to ```let``` so they're immutable.